app/src/main/java/com/mapzen/erasermap/AndroidModule.java

```diff
@@ -116,7 +126,7 @@ public AndroidModule(EraserMapApplication application) {
 
             @Override public Map<String, String> queryParamsForRequest() {
                 Map<String, String> params = new HashMap<>();
-                params.put(ApiKeyConstants.API_KEY, apiKeys.getApiKey());
+                params.put(Http.PARAM_API_KEY, apiKeys.getApiKey());
```

Comment:
Maybe requires more changes (than we want with EM), but probably we should use sceneUpdate with sceneLoad for consistency with SDK usage.